.gitlab-ci.yml

```diff
@@ -93,7 +99,7 @@ Review_db:WP-HX:
     extends: .deploy-review
     environment:
         name: wp-hx-dev-ing
-        url: http://update-track-api.review.ensembl.org
+        url: http://update.review.ensembl.org
```

Comment:
@veidenberg  Don't forget to remove this before merging

Reply:
With all the reviews apps in gitlab interface will lead to update.review.ensembl.org. Either it should be $CI_COMMIT_REF_SLUG.review.ensembl.org or just remove this.

Reply:
This is (at least meant to be) a specific case of review apps that is connected to dev db (not live db like default review apps) and it's deployed only from branch `update`, so it's URL is fixed.

Reply:
For other review apps, there is `Review:WP-HX` on line 86.

Reply:
I'm afraid I still cannot follow. What is the preferred way to do this (i.e. can you elaborate "specifying internal database host for whatever review app")? In current case, I've set a specific feature branch (`update`) to differentiate it from other revoew apps and to load the configmap for using internal dev db through ci-cd.

Reply:
The `Review_db` job was here already before (also present on main branch). I only fixed the URL because the previous fixed URL was incorrect. Are there any benefits of opening a separate PR to fix (again) the URL?